platform/engineering/design-docs/2021-06-11-small-batch-publishing.md

```diff
@@ -0,0 +1,350 @@
+| Check collections                                         | X              |                            |                         |                                                                                                                                                         |
+| Group collections                                         | X              | X                          | X                       |                                                                                                                                                         |
+| Reset left rail navigation menu levels                    | X              | X                          | X                       |                                                                                                                                                         |
+| Add the date to filenames                                 | X              |                            |                         |                                                                                                                                                         |
+| Add app assets                                            | X              |                            |                         |                                                                                                                                                         |
+| Add content assets                                        | X              |                            |                         |                                                                                                                                                         |
+| Plug the content into the templates                       | X              | X                          | X                       |                                                                                                                                                         |
+| Translate the markdown to html                            | X              | X                          |                         | This probably isn’t needed on SBP since the markdown pages will not be processed.                                                                       |
+| Add permalinks and change foo.md to foo/index.html        | X              | X                          |                         |                                                                                                                                                         |
+| Create header and footer                                  | X              |                            |                         |                                                                                                                                                         |
```

Comment:
There's an open ticket to add this to the preview functionality

Reply:
Would this requre the https://github.com/department-of-veterans-affairs/vagov-content repo?

Reply:
TBD on that I think @theodur is picking up that ticket next week

Reply:
Is this the issue? https://github.com/department-of-veterans-affairs/va.gov-team/issues/25793

Reply:
@indytechcook yeah that's the one

Reply:
Thanks, I linked it to the doc!